components/ws-proxy/pkg/sshproxy/forward.go

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package sshproxy
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"sync"
+	"time"
+
+	"golang.org/x/crypto/ssh"
+)
+
+const GITPOD_USERNAME = "gitpod"
```

Comment:
nit: Go conventionally uses MixedCaps also for constants (see https://go.dev/doc/effective_go#mixed-caps)